praw/models/reddit/submission.py

```diff
@@ -191,6 +191,30 @@ def unhide(self, other_submissions=None):
         for submissions in self._chunk(other_submissions, 50):
             self._reddit.post(API_PATH['unhide'], data={'id': submissions})
 
+    def crosspost(self, subreddit, title=None, send_replies=True):
+        """Crosspost the submission to a subreddit.
+
+        :param subreddit: Name of the subreddit or :class:`~.Subreddit`
+            object to crosspost into.
+        :param title: Title of the submission. Will use this submission's
+            title if `None`. (default: None).
```

Comment:
Minor grammar suggestions for consistency:

* Omit the period before the `(default: None)`, as is done in the `send_replies comment.
* Add a period after `submission` on the `returns` description.